.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I am not sure if `npx` will work in the same way, but since we are using `yarn`, I think we use [`yarn dlx`](https://yarnpkg.com/cli/dlx) here.

Reply:
```suggestion
yarn dlx lint-staged
```